guides/common/attributes-satellite.adoc

```diff
@@ -155,7 +155,7 @@
 // Used in downstream guides
 :RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
 
-:project-client-name: Satellite Client 6
+:project-client-name: Red Hat Satellite Client 6
 :project-client-RHEL7-url: {RepoRHEL7ServerSatelliteToolsProjectVersion}
 :project-client-RHEL8-url: {RepoRHEL8ServerSatelliteToolsProjectVersion}
```

Comment:
To make it even more confusing, this attribute is the reverse of `project-client-name`: in upstream it's the URL where in downstream it's the name. I'm starting to get a feeling that we shouldn't attempt to reuse the attributes between upstream and downstream since they're fundamentally different and used in different contexts.

What if we undefine `project-client-RHEL7-url` in this guide (you can do that with asciidoc, right?) to make sure it errors out if it's used. Then simply use `RepoRHEL7ServerSatelliteToolsProjectVersion` where it is needed.

Reply:
I would actually prefer to remove the `*rhel*-url`s altogether and replace them with the Sat attributes. They're only used in 3 places in total.

Reply:
Well, that's sort of what I was suggesting. Except I had still assumed we needed them upstream. But you're right that the `rhel` part doesn't make sense since in upstream we prefer `el`.

Reply:
-> #3160